tests/basic/storage_pool_test.go

```diff
@@ -2053,4 +2052,574 @@ var _ = Describe("{VolUpdateAddDrive}", func() {
+			dash.VerifyFatal(resizeErr, nil, fmt.Sprintf("Verify pool [%s] on node [%s] expansion using auto", rebootPoolID, restartDriver.Name))
+
+		}
+		opts := make(map[string]bool)
+		opts[scheduler.OptionsWaitForResourceLeakCleanup] = true
+		ValidateAndDestroy(contexts, opts)
+
+		Step("destroy all the applications created before test runs", func() {
+			opts := make(map[string]bool)
+			opts[scheduler.OptionsWaitForResourceLeakCleanup] = true
+			for _, ctx := range contexts {
+				TearDownContext(ctx, opts)
+			}
+		})
+	})
+
+	JustAfterEach(func() {
+		defer EndTorpedoTest()
+		AfterEachTest(contexts, testrailID, runID)
+	})
+})
+
+var _ = Describe("{PoolIncreaseSize20TB}", func() {
+	var testrailID = 51292
+	// Testrail Description : Resize a pool of capacity of 100GB to 20TB
+	// Testrail Corresponds : https://portworx.testrail.net/index.php?/cases/view/51292
+	var runID int
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("PoolIncreaseSize20TB", "Resize a pool of capacity of 100GB to 20TB", nil, testrailID)
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+	})
+
+	var contexts []*scheduler.Context
+	//var vol_ids []string
+	stepLog := "should get the existing storage node and expand the pool by resize-disk"
+	It(stepLog, func() {
+
+		contexts = make([]*scheduler.Context, 0)
+
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("snapcreateresizepool-%d", i))...)
+		}
+		ValidateApplications(contexts)
+
+		pools, err := Inst().V.ListStoragePools(metav1.LabelSelector{})
+		log.FailOnError(err, "Failed to list storage pools")
+		dash.VerifyFatal(len(pools) > 0, true, "Storage pools exist ?")
+
+		// pick a pool from a pools list and resize it
+		poolIDToResize, err := GetPoolIDWithIOs()
+		log.FailOnError(err, "error identifying pool to run test")
+		dash.VerifyFatal(len(poolIDToResize) > 0, true, fmt.Sprintf("Expected poolIDToResize to not be empty, pool id to resize [%s]", poolIDToResize))
+
+		poolToBeResized := pools[poolIDToResize]
+		dash.VerifyFatal(poolToBeResized != nil, true, "Pool to be resized exist?")
+
+		// px will put a new request in a queue, but in this case we can't calculate the expected size,
+		// so need to wain until the ongoing operation is completed
+		stepLog = "Verify that pool resize is not in progress"
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			if poolResizeIsInProgress(poolToBeResized) {
+				// wait until resize is completed and get the updated pool again
+				poolToBeResized, err = GetStoragePoolByUUID(poolIDToResize)
+				log.FailOnError(err, "Failed to get pool using UUID [%s]", poolIDToResize)
+			}
+		})
+
+		var expectedSize uint64
+		var expectedSizeWithJournal uint64
+
+		// Marking the expected size to be 2TB
+		expectedSize = (2048 * 1024 * 1024 * 1024 * 1024) / units.TiB
+
+		stepLog = "Calculate expected pool size and trigger pool resize"
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
```

Comment:
Same as above `Step("Calculate expected pool size and trigger pool resize", func() {}`, don't need 3 lines of same thing

Reply:
in that case we need 3 lines to be present